ms/MSOper/MSMetaData.cc

```diff
@@ -4999,6 +4999,21 @@ std::shared_ptr<vector<int>> MSMetaData::_almaReceiverBands(uint nspw) const {
     return freqBands;
 }
 
+std::vector<std::vector<uInt>> MSMetaData::_getSpwToPolMap() const {
+    if (! _spwIDToPolIDMap.empty()) {
+        return _spwIDToPolIDMap;
+    }
+    const auto spwPolToDDID = getSpwIDPolIDToDataDescIDMap();
+    _spwIDToPolIDMap.resize(nSpw(true));
+    for (const auto &p : spwPolToDDID) {
+        _spwIDToPolIDMap[p.first.first].push_back(p.first.second);
```

Comment:
Would be a bit more readable as
```suggestion
        uInt spwID = p.first.first;
        uInt polId = p.first.second;
        _spwIDToPolIDMap[spwId].push_back(polId);
```